core/app/models/spree/transfer_item.rb

```diff
@@ -51,8 +51,12 @@ def stock_transfer_closed?
       stock_transfer.closed?
     end
 
+    def stock_transfer_shipped?
+      stock_transfer.shipped?
```

Comment:
Yeah, both `stock_transfer_closed?` and `stock_transfer_shipped?` should be inlined then.. I'm into it.